java/src/main/java/com/saucelabs/saucebindings/DataCenter.java

```diff
@@ -3,9 +3,11 @@
 import lombok.Getter;
 
 public enum DataCenter {
-    US_WEST("ondemand.us-west-1.saucelabs.com"),
-    US_EAST("ondemand.us-east-1.saucelabs.com"),
-    EU_CENTRAL("ondemand.eu-central-1.saucelabs.com");
+    US_LEGACY("https://ondemand.saucelabs.com/wd/hub"),
+    US_WEST("https://ondemand.us-west-1.saucelabs.com/wd/hub"),
+    US_EAST("https://ondemand.us-east-1.saucelabs.com/wd/hub"),
+    HEADLESS("https://ondemand.us-east-1.saucelabs.com/wd/hub"),
```

Comment:
@titusfortner do we need this, I think it's going to die, right?

Reply:
nope, it's sticking around according to Matt.